desktop/dashboard.md

```diff
@@ -125,10 +108,99 @@ Click on a specific container for detailed information about the container. The
+## Explore your images
+
+The **Images**  view is a simple interface that lets you manage Docker images without having to use the CLI. By default, it displays a list of all Docker images on your local disk. To view images in remote repositories, click **Sign in** and connect to Docker Hub. This allows you to collaborate with your team and manage your images directly through Docker Desktop.
+
+The Images view allows you to perform core operations such as running an image as a container, pulling the latest version of an image from Docker Hub, pushing the image to Docker Hub, and inspecting images.
+
+In addition, the Images view displays metadata about the image such as the tag, image ID, date when the image was created, and the size of the image. It also displays **In Use** tags next to images used by running containers. This allows you to review the list of images and use the **Clean up images** option to remove any unused and dangling images from the disk to reclaim space. The Images view also allows you to search images on your local disk and sort them using various options.
+
+Let's explore the various options in the **Images** view.
+
+If you don’t have any images on your disk, run the command `docker pull redis` in a terminal to pull the latest Redis image. This command pulls the latest Redis image from Docker Hub.
+
+Select **Dashboard** > **Images** to see the Redis image.
+
+![Redis image](images/redis-image.png){:width="700px"}
+
+### Run an image as a container
+
+Now that you have a Redis image on your disk, let’s run this image as a container:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the Redis image from the list and click **Run**.
+3. When prompted, click the **Optional settings** drop-down to specify a name, port, volumes, and click **Run**.
+
+    To use the defaults, click **Run** without specifying any optional settings. This creates a new container from the Redis image and opens it on the **Container/Apps** view.
+
+### Pull the latest image from Docker Hub
+
+To pull the latest image from Docker Hub:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the image from the list and click the more options button.
+3. Click **Pull**. This pulls the latest version of the image from Docker Hub.
+
+> **Note**
+>
+> The repository must exist on Docker Hub in order to pull the latest version of an image. You must be logged in to pull private images.
+
+### Push an image to Docker Hub
+
+To push an image to Docker Hub:
+
+1. From the Docker menu, select **Dashboard** > **Images**. This displays a list of images on your local disk.
+2. Select the image from the list and click the more options button.
+3. Click **Push to Hub.**
+
+> **Note**
+>
+> You can only push an image to Docker Hub if the image belongs to your Docker ID or your organization.
```

Comment:
What does "belongs to" mean? I believe it means it has the correct username/organisation name in its tag, but it's unclear here.